### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by mattisonzhao**
I found several refactorable code when I read the code.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@mattisonzhao Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=18736)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-boot&pullRequestId=18736) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@mattisonzhao Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=18736)!

File: spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/web/PathMappedEndpoints.java [hunk 1]
```diff
@@ -65,13 +65,13 @@ public PathMappedEndpoints(String basePath, Collection<EndpointsSupplier<?>> sup
 
 	private Map<EndpointId, PathMappedEndpoint> getEndpoints(Collection<EndpointsSupplier<?>> suppliers) {
 		Map<EndpointId, PathMappedEndpoint> endpoints = new LinkedHashMap<>();
-		suppliers.forEach((supplier) -> {
+		for (EndpointsSupplier<?> supplier : suppliers) {
```
Comment (ehabqadah) [inline spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/web/PathMappedEndpoints.java]: Why not replacing the both `forEach` loops?  if there is a benefit of doing that (I might think of performance reasons)

Comment (philwebb) [reply spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/web/PathMappedEndpoints.java]: There's often a GC impact when using streams, but I think it's pretty safe to use `forEach` here. I'm not sure why we'd want to change it.

File: spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/http/ApiVersion.java [hunk 2]
```diff
@@ -75,7 +75,7 @@ private static ApiVersion forType(String type) {
 			try {
 				return valueOf(type.toUpperCase());
 			}
-			catch (IllegalArgumentException ex) {
+			catch (IllegalArgumentException ignored) {
```
Comment (philwebb) [inline spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/http/ApiVersion.java]: We always use `ex` for caught exceptions.

File: spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/SimpleStatusAggregator.java [hunk 3]
```diff
@@ -38,7 +38,7 @@
 
 	private static final List<String> DEFAULT_ORDER;
 	static {
-		List<String> defaultOrder = new ArrayList<String>();
+		List<String> defaultOrder = new ArrayList<>();
```
Comment (philwebb) [inline spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/SimpleStatusAggregator.java]: All other changes look good. Thanks!

**Comment by snicoll** [comment]
@mattisonzhao thank you for making your fist contribution to Spring Boot.

**Pull request closed without merging**